.vscode/launch.json

```diff
@@ -160,10 +179,13 @@
             "request": "launch",
             "program": "${workspaceFolder}/node_modules/.bin/ts-mocha",
             "args": ["--forbid-only=false", "--forbid-pending=false", "${file}"],
-            "outFiles": ["${workspaceFolder}/**/out/**/*.(m|c|)js", "!**/node_modules/**"],
-            "console": "integratedTerminal",
-            "pauseForSourceMap": true,
-            "stopOnEntry": true
+            "outFiles": [
+                "${workspaceFolder}/**/src/**/*.ts",
+                "${workspaceFolder}/**/out/**/*.ts",
```

Comment:
Why would you need `ts files here, both `src` and `out`?

Reply:
The debugger is using these files.  The out/**/*.js.map files alone didn't seem to work for breakpoints with ts-mocha.  After adding those lines the breakpoints started working across both tests and regular .ts files.  I didn't really have time to debug the debugger, so I just left them because being able to set breakpoints is what I needed.